lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tests for Progress Tab API in the Course Home API
+"""
+
+from datetime import datetime
+import ddt
+
+from django.urls import reverse
+
+from course_modes.models import CourseMode
+from lms.djangoapps.course_home_api.tests.utils import BaseCourseHomeTests
+from lms.djangoapps.course_home_api.toggles import COURSE_HOME_MICROFRONTEND
+from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from openedx.core.djangoapps.user_api.preferences.api import set_user_preference
+from student.models import CourseEnrollment
+from student.tests.factories import UserFactory
+
+
+@COURSE_HOME_MICROFRONTEND.override(active=True)
+@ddt.ddt
+class ProgressTabTestViews(BaseCourseHomeTests):
+    """
+    Tests for the Progress Tab API
+    """
+    def setUp(self):
+        super().setUp()
+        self.url =  reverse('course-home-progress-tab', args=[self.course.id])
+        ContentTypeGatingConfig.objects.create(enabled=True, enabled_as_of=datetime(2017, 1, 1))
+
+    @ddt.data(CourseMode.AUDIT, CourseMode.VERIFIED)
+    def test_get_authenticated_enrolled_user(self, enrollment_mode):
+        CourseEnrollment.enroll(self.user, self.course.id, enrollment_mode)
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+
+        # Pulling out the date blocks to check learner has access.
+        self.assertNotEqual(response.data['courseware_summary'], None)
```

Comment:
I think there is an `assertNotNone` method you can use here.

Reply:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone